docs/src/main/sphinx/connector/iceberg.md

```diff
@@ -865,8 +865,8 @@ connector using a {doc}`WITH </sql/create-table-as>` clause.
   - Optionally specifies the file system location URI for the table.
 * - `format_version`
   - Optionally specifies the format version of the Iceberg specification to use
-    for new tables; either `1` or `2`. Defaults to `2`. Version `2` is required
-    for row level deletes.
+    for new tables; either `1`, `2` or `3`. Defaults to `2`. Only version `2`
```

Comment:
I didn't get what exactly is the user facing impact of this change.
I see that we disallow altering table to v3 and update/merge.
Are reads, optimize and inserts to existing v3 tables allowed ? Is it valid to allow them while we don't have support for v3 deletion vectors ? Do we error out only if a v3 deletion vector is encountered ?
I think it would be nicer if we just added support for deletion vector along with v3 support, if not sure what's the benefit of supporting v3 without that.

Reply:
@raunaqmorarka See PR description. It's a prepartory task for other V3 spec.